functions.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+def get_pages(count, limit):
```

Comment:
These are utility functions... a convention is to name the file `utils` instead of `functions`

Reply:
This has been updated now.